Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-stretch
+
+WORKDIR /opt/avere-europarl
+
+RUN pip3 install Scrapy==1.6.0
+
+ADD LICENSE README.md ./
```

Comment:
You don't need text files in your docker image.

You can simplify this whole section by just:
```
RUN pip3 install ...
RUN mkdir /app
WORKDIR /app
ADD . .
CMD python3 /app/scraper/main.py
```

You could also only add the `scraper` subfolder to your Dockerfile, like so: `ADD scraper .`. This should be OK since you only have python code inside it.